base/shell/explorer/trayprop.cpp

```diff
@@ -279,8 +279,18 @@ PropSheetProc(HWND hwndDlg, UINT uMsg, LPARAM lParam)
     {
         case PSCB_INITIALIZED:
         {
+            RECT pos;
+            RECT rcWork;
             hIcon = LoadIconW(hExplorerInstance, MAKEINTRESOURCEW(IDI_STARTMENU));
             SendMessageW(hwndDlg, WM_SETICON, ICON_BIG, (LPARAM)hIcon);
+
+            GetWindowRect(hwndDlg, &pos);
+            SystemParametersInfo(SPI_GETWORKAREA, 0, &rcWork, 0);
+            if (pos.bottom > rcWork.bottom)
+                pos.top -= (pos.bottom - rcWork.bottom);
+            if (pos.right > rcWork.right)
+                pos.left -= (pos.right - rcWork.right );
```

Comment:
```suggestion
                pos.left -= (pos.right - rcWork.right);
```
Space is still here...